### PR TITLE
Page with the installation versions

### PR DESCRIPTION
**Pull request by hascar**
Implemented page with the installation versions
The tests are green here: https://ci.openquake.org/job/zdevel_oq-platform2/200/

### CODE REVIEW

File: verifier-guest.sh [hunk 1]
```diff
@@ -31,6 +31,7 @@ geonode_setup_env()
 
 ##function nested applicatione
```
Comment (nastasi-oq) [inline verifier-guest.sh]: fix typo

File: verifier-guest.sh [hunk 2]
```diff
@@ -31,6 +31,7 @@ geonode_setup_env()
...
 nested_exec() {
```
Comment (nastasi-oq) [inline verifier-guest.sh]: use a more proper function name

File: openquakeplatform/test/versions_test.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    def page_app_versions_test(self):
+
+        versions_url = '/versions'
+
+        # redirect page of versions application
+        pla.get(versions_url)
+
+        pla.xpath_finduniq(
+            "//h2[@class='page-title'"
+            " and normalize-space(text())='Application Versions']")
```
Comment (nastasi-oq) [inline openquakeplatform/test/versions_test.py]: Please check that at least one of the plugin application is recognized by the page

File: verifier-guest.sh (hunk 1 above)
Comment (hascar) [reply verifier-guest.sh]: Fixed here: https://github.com/gem/oq-platform2/pull/20/commits/7e8f0b3c2911de772b5b72100335ca79cf9e1250

File: verifier-guest.sh (hunk 2 above)
Comment (hascar) [reply verifier-guest.sh]: Fixed here: https://github.com/gem/oq-platform2/pull/20/commits/7e8f0b3c2911de772b5b72100335ca79cf9e1250

File: openquakeplatform/test/versions_test.py (hunk 3 above)
Comment (hascar) [reply openquakeplatform/test/versions_test.py]: Fixed here: https://github.com/gem/oq-platform2/pull/20/commits/ea629723cc940b2ca3bfb3be890adee142d7db3b

**Pull request merged**